images/refinerycms-images.gemspec

```diff
@@ -20,7 +20,8 @@ Gem::Specification.new do |s|
   s.test_files        = `git ls-files -- spec/*`.split("\n")
 
   s.add_dependency 'dragonfly',               '~> 1.0.0'
-  s.add_dependency 'globalize',               ['>= 4.0.0', '< 5.2']
+  s.add_dependency 'globalize',               '5.1.0.beta1'
```

Comment:
this should probably specify the beta as the minimum, i.e. `['~> 5.1.0', '>= 5.1.0.beta1']`?